roles/edpm_neutron_metadata/templates/wrappers/haproxy.j2

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+{% if edpm_neutron_metadata_agent_sidecar_debug -%}
+set -x
+{%- endif %}
+
+ARGS="$@"
+
+IMAGE_NAME={{ edpm_neutron_metadata_agent_sidecar_haproxy_image_name }}
+NEUTRON_CONFIG_DIR={{ edpm_neutron_metadata_agent_config_dir }}
+NEUTRON_LIB_DIR={{ edpm_neutron_metadata_agent_lib_dir }}
+
+{% raw -%}
+# Extract the network namespace UUID from the command line args provided by
+# neutron. Typically of the form (with dnsmasq as an example):
+#
+# dnsmasq --no-hosts --no-resolv --except-interface=lo \
```

Comment:
the comment is not valid for reasons I listed elsewhere (no CLI args are checked to extract namespace name), but here also because you list dnsmasq in haproxy wrapper.